### PR TITLE
Add publish release script.

### PR DESCRIPTION
**Pull request by samelhusseini**
Add a release script that:
- Logs into the wombot npm registry
- Clears the /dist directory if it exists
- Clones a fresh copy of blockly-samples/master under /dist
- (dist is already gitignored)
- Runs npm install
- Run npm publish:check (if its a dry run)
- Run npm publish:release (if its not a dry run)

Unrelated: Update license script to run for all plugins.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -8,17 +8,92 @@
...
+    }
+
+    // Clone a fresh copy of blockly-samples.
+    console.log(`Checking out a fresh copy of blockly-samples under\
+  ${path.resolve(releaseDir)}`);
+    execSync(`git clone https://github.com/google/blockly-samples ${releaseDir}`,
+        {stdio: 'pipe'});
+
+    // Run npm install.
+    console.log('Running npm install.');
```
Comment (alschmiedt) [inline gulpfile.js:68]: I think we might also want to run npm run boot

File: package.json [hunk 2]
```diff
@@ -13,14 +13,17 @@
         "postinstall": "npm run boot",
         "test": "lerna run test",
         "publish:check": "lerna changed",
-        "publish": "lerna publish"
+        "publish:release": "lerna publish",
```
Comment (alschmiedt) [inline package.json:16]: Do we want to keep this one around, if we are always planning on going through the publish script?

**Review by alschmiedt — APPROVED** [review]
lgtm once comments are addressed

File: gulpfile.js (hunk 1 above)
Comment (samelhusseini) [reply gulpfile.js:68]: Npm install runs boot.

Comment (samelhusseini) [reply gulpfile.js:68]: As a postinstall step.

File: package.json (hunk 2 above)
Comment (samelhusseini) [reply package.json:16]: The publish script calls ``publish:release``.

Comment (samelhusseini) [reply package.json:16]: https://github.com/google/blockly-samples/pull/247/files#diff-b9e12334e9eafd8341a6107dd98510c9R72

**Review by alschmiedt — APPROVED** [review]

**Pull request merged**